testing_local_changes.md

```diff
@@ -0,0 +1,115 @@
+# Background
+
+edpm-ansible is included in the openstack-ansibleee-runner container image,
+which is key component used by the dataplane-operator that deploys EDPM
+nodes. The dataplane-operator's CRD includes support for specifying additional
+volume mounts for the ansibleee pods, which provides a mechanism for accessing
+a local copy of edpm-ansible. This makes it possible to develop and test local
+changes edpm-ansible without having to build and deploy a new
+openstack-ansibleee-runner container image.
+
+# Provide NFS access to your edpm-ansible directory
+
+The technique described here uses NFS to access the edpm-ansible directory on
+your development system, so you'll need to install an NFS server and create
+an appropriate export on your development system. Of course, this implies
+your OpenShift deployment that runs the dataplane-operator has access to
+the NFS server, including any required firewall rules.
+
+* [EL 8 instructions](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/exporting-nfs-shares_deploying-different-types-of-servers#assembly_configuring-the-nfs-server-to-run-behind-a-firewall_exporting-nfs-shares)
+* [EL 9 instructions](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_file_systems/exporting-nfs-shares_managing-file-systems#assembly_configuring-the-nfs-server-to-run-behind-a-firewall_exporting-nfs-shares)
+
+When using OpenShift Local (aka CRC), your export will be something like this:
+
+```
+% cat &lt;&lt;EOF &gt;/etc/exports
```

Comment:
is it `cat <<EOF >/etc/exports` ?

Reply:
It will be when viewed as html. The problem with markdown is '<' characters get confused with html tags (even in literal code blocks), and using "&lt;" allows the page to be rendered correctly.

But I'll follow your suggestion and move this to an rst file in the docs area, where the conversion to html isn't an issue.